code/game/machinery/doors/blast_door.dm

```diff
@@ -22,6 +22,8 @@
 	var/icon_state_closing = null
 	var/damage = BLAST_DOOR_CRUSH_DAMAGE
 
+	layer = 3.3
+	open_layer = 3.3
```

Comment:
What is the purpose of this change?

Reply:
Shutters wouldn't overlay over glass windows, instead, glass windows would overlay over shutters.